app/models/user.rb

```diff
@@ -2,4 +2,9 @@ class User < ApplicationRecord
   include GDS::SSO::User
 
   serialize :permissions, Array
+
+  enum :role, {
+    super_admin: "super_admin",
```

Comment:
A bit late to the party and this is just a question rather than an objection,  but instead of `super_admin` was `system_admin` e.g. us as system administrators, and `org_admin`, e.g. admins for an org to manage their own users considered? Those names have more distinction to me because super_admin isn't immediately clear what scope it has, could be a super user on the org for example, but since its easily clarified with some knowledge sharing its not a big deal.

This is based on my understanding that `super_admin` will eventually just be us and `admin` will be used for org admins?

Reply:
We could even make it _really_ clear who these users are by calling it `forms_service_admin` or `service_admin`

Reply:
Team decided to review this at another time but I do agree with you that system_admin is a bit clearer what take of admin is actually being done.